src/Components/Settings/AddAdminPopup.jsx

```diff
@@ -45,6 +45,13 @@ const AddAdminPopup = ({ setAdminPopup, setFlag }) => {
               "\n" +
               "Member has been invited successfully.",
           });
+          //////////////////////////
```

Comment:
Role created

Reply:
@soumitra-quest when the "owner" invites a new team member, do we create a new user?
How should I get the id of the new user as well id of the "owner" who invited the new user?

I am thinking the code will look like this - 

```
  const userPermission_ = {
          role: "Admin",
          user_id: "",
          assigned_by: ""
        }
        generalFunction.addUserPermission(userPermission_);
          setFlag((prev) => !prev);
          setTimeout(function () {
            generalFunction.hideLoader();
          }, 500);
        }
      })
```